### DISCUSSION TITLE
white screen  encounter when opening the Edit status modal

### DISCUSSION THREAD

**Discussion by Abhimishra2424** [community/community#60473] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello 
The white screen encounter when opening the Edit status modal.
Go To Profile > edit status

**Comment by ghostinhershell** [comment]
Hi @Abhimishra2424, we're happy you're here! You are more likely to get a useful response if you are explicit about the issue that you're seeing. Giving a few more details might help someone give you a nudge in the right direction.

**Comment by RoofaE** [comment]
Any extra/clear information about this topic? I can try my best to help

**Comment by sandroamiridze** [comment]
Here are a few steps you can try to troubleshoot the issue:

    Clear Cache and Cookies: Sometimes, cached data can cause unexpected behavior. Clear your browser's cache and cookies and then try opening the "Edit status" modal again.

    Use a Different Browser: If the issue persists, try using a different web browser to access GitHub. This can help determine if the problem is specific to your browser.

    Disable Browser Extensions: Browser extensions can sometimes interfere with the functioning of websites. Try disabling any browser extensions and see if that resolves the white screen issue.

    Check Network Connection: A stable internet connection is crucial for accessing web applications. Make sure you have a stable and active internet connection.

    Check GitHub Status: GitHub occasionally experiences outages or issues. You can check the GitHub status page (status.github.com) to see if there are any ongoing problems that might be causing the white screen issue.

    Contact GitHub Support: If none of the above steps resolve the issue, you might want to contact GitHub support for assistance. They can provide you with specific guidance and solutions for the problem you're facing.

**Comment by cehfisher** [comment]
Hi @Abhimishra2424 - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Reply by Abhimishra2424** [reply]
ok ma'am
